apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -318,6 +366,25 @@
             <tr>
               <td class="text-nowrap"><%= row.formatted_date %></td>
               <td><%= row.auth_user_uid %></td>
+              <td><a tabindex="0" class="btn btn-xs btn-secondary"
```

Comment:
Rows where we don't have a `client_fingerprint_id` are still showing the button (just very thin) and have an empty popover (see screenshot below). I think rows like this should just show an `&mdash;` instead without a button.

<img width="210" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/875833/481346f4-b334-4b45-906e-38710797734a">

Reply:
This has now been updated to only display an `&mdash;` when there is no client fingerprint to display.